nodejs/examples/rabbitmq.js

```diff
@@ -0,0 +1,30 @@
+const amqp = require('amqplib/callback_api');
+const config = require("platformsh-config").config();
+
+exports.usageExample = async function() {
+    try {
+        const credentials = config.credentials('rabbitmq');
+        console.log(credentials);
+
+        let connectionString = 'amqp://' + credentials.username + ':' + credentials.password + '@' + credentials.host + ':' + credentials.port;
+        console.log(connectionString);
+
+        let connection = await amqp.connect(connectionString);
+        console.log(connection);
+
+        let channel = await connection.createChannel();
+        console.log(channel);
+
+        let queue = 'hellotest';
+        let message = 'Deploy Friday!';
+
+        await channel.assertQueue(queue, {durable: false});
+        await channel.sendToQueue(queue, Buffer.from(message))
+
+        let output = 'Message sent: ' + queue + ' ' + message;
+
+        return output;
+    } catch (error) {
```

Comment:
Everything is supposed to run just fine in this example, so you might want to get rid of the `try`/`catch` just to remove some noise around the code example.

I don't feel strongly about this though.

Just for the sake of completion, `async`/`await` is good but not always clearer. It feels like writing synchronous code when things are async so it looks simple and familiar but has some caveats e.g. control flow and error handling. This `catch` here would catch any error happening in the try block so it makes it hard to have a good error handling mechanism (What broke? The connection? A non-promise-related exception got thrown? Something else?).
It's okay to use a `.then(...)` approach if that makes things simpler to reason about. Whether or not things are actually "simpler" is pretty subjective though.